gef-ui/server/models/application.js

```diff
@@ -62,7 +62,7 @@ class Application {
     try {
       const application = await getApplication(id);
 
-      if (application.bank.id !== user.bank.id) {
+      if (application.bank.id !== user.bank.id && !user.roles.includes('admin')) {
```

Comment:
was it just this line that we needed to get admins to see the deals?

Reply:
Yes. Admin doesn't have the matching bank id. So this line should stop admins returning null so they can see the deals and stop other banks from opening other banks deals